sklearn/utils/estimator_checks.py

```diff
@@ -3147,3 +3157,18 @@ def check_n_features_in_after_fitting(name, estimator_orig):
 
     with raises(ValueError, match=msg):
         estimator.partial_fit(X_bad, y)
+
+
+def check_estimator_get_tags_default_keys(name, estimator_orig):
+    # check that if _get_tags is implemented, it contains all keys from
+    # _DEFAULT_KEYS
+    estimator = clone(estimator_orig)
+    if not hasattr(estimator, "_get_tags"):
+        return
```

Comment:
We probably need a test to cover this line which is not covered according to codecov.

I know that this will be covered once we run the full `check_estimator` on the minimal estimators, but in the mean time we could just run `check_estimator_get_tags_default_keys` on one of them.